services/frontend/pkg/revaconfig/config.go

```diff
@@ -1,24 +1,39 @@
 package revaconfig
 
 import (
+	"bufio"
 	"net/url"
+	"os"
 	"path"
+	"path/filepath"
 	"strconv"
 	"time"
 
+	"github.com/owncloud/ocis/v2/ocis-pkg/config/defaults"
+	"github.com/owncloud/ocis/v2/ocis-pkg/log"
 	"github.com/owncloud/ocis/v2/ocis-pkg/version"
 	"github.com/owncloud/ocis/v2/services/frontend/pkg/config"
 )
 
 // FrontendConfigFromStruct will adapt an oCIS config struct into a reva mapstructure to start a reva service.
-func FrontendConfigFromStruct(cfg *config.Config) (map[string]interface{}, error) {
+func FrontendConfigFromStruct(cfg *config.Config, logger log.Logger) (map[string]interface{}, error) {
 	webURL, err := url.Parse(cfg.PublicURL)
 	if err != nil {
 		return nil, err
 	}
 	webURL.Path = path.Join(webURL.Path, "external")
 	webOpenInAppURL := webURL.String()
 
+	var bannedPasswordsList map[string]struct{}
+	if cfg.PasswordPolicy.BannedPasswordsList != "" {
+		bannedPasswordsList, err = readMultilineFile(cfg.PasswordPolicy.BannedPasswordsList)
+		if err != nil {
+			logger.Err(err).
+				Str("server", cfg.Service.Name).
+				Msgf("Failed to load the banned passwords from a file %s, skipped.", cfg.PasswordPolicy.BannedPasswordsList)
```

Comment:
We should return the error here. If someone wants to set a banned password list, they must immediately know that it didn't work.